src/index.html

```diff
@@ -5,8 +5,81 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>基本コーディング</title>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick-theme.min.css">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.min.css">
 </head>
 <body>
-
+  <div class="newrequest">
```

Comment:
[imo]

section タグが適切かと思います！

```suggestion
  <section class="newrequest">
```

Reply:
セクションタグに変更しました。